src/main/java/com/redhat/rcm/offliner/Options.java

```diff
@@ -62,6 +64,11 @@
     @Option( name = "-s", aliases = { "--mavensettings" }, metaVar = "FILE", usage = "Path to settings.xml used when a pom is used as the source file" )
     private File settingsXml;
 
+    @Option( name = "-s", aliases = { "--maventypemapping" }, metaVar = "MAPPING", usage = "List of mapping where key is "
+        + "type and value is file extension with or without classifier separated by colon. List elements are separated by "
+        + "semicolons." )
+    private String typeMapping = DEFAULT_TYPE_MAPPING;
+
```

Comment:
is this designed to have the full type mapping provided on the command line? If so, using a file reference instead might make these definitions more portable / maintainable